src/GeositeFramework/plugins/launchpad/style.css

```diff
@@ -26,3 +26,41 @@
     list-style-type: none;
     padding: 0;
 }
+
+.launchpad-plugin .plugins .button.truncate-toggle, .scenarios .button.truncate-toggle {
+    padding: 2px !important;
+    font-size: 0.7em !important;
+    width: 35%;
+    float: none;
+}
+
+.truncate-box {
+    overflow: hidden;
+    position: relative;
+    line-height: 1.2em;
+    max-height: 2.6em;
```

Comment:
The text itself isn't truncated, the box class is conditioned on the text length. The visible height is the mechanism of hiding the text.